adserver/tasks.py

```diff
@@ -463,6 +463,15 @@ def update_previous_day_reports(day=None):
     daily_update_uplift(start_date)
     daily_update_regiontopic(start_date)
 
+    # Send notification to Slack about previous day's reports
+    if settings.SLACK_TOKEN:
```

Comment:
Might be worth moving this the `slack_message` code, so we don't need it everywhere?

Reply:
Cool. It's a third party package but I'll move a wrapper into utils.